CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+
+### Updating from v0.7.0
+
+#### Cluster update steps
+
+> **NOTE:** Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a version older than `v0.7.0`, update to `v0.7.0` first and only then proceed with the update to `v0.8.0`.
+
+Execute the following steps in your cluster configuration directory:
+
+1. Download and install the lokoctl binary by following the
+  [v0.8.0 installation guide](https://github.com/kinvolk/lokomotive/blob/v0.8.0/docs/installer/lokoctl.md)
+  and verify the version using `lokoctl version`:
+
+  ```bash
+  v0.8.0
+  ```
+
+2. On all platforms **except AKS**, update Calico CRDs:
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/bgpconfigurations.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/bgppeers.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/blockaffinities.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/clusterinformations.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/felixconfigurations.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/globalnetworkpolicies.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/globalnetworksets.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/hostendpoints.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamblocks.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamconfigs.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ipamhandles.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/ippools.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/kubecontrollersconfigurations.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/networkpolicies.yaml
+  kubectl apply -f https://raw.githubusercontent.com/kinvolk/lokomotive/release-v0.8.0/assets/charts/control-plane/calico/crds/networksets.yaml
```

Comment:
This does not look good to look at honestly

Reply:
I agree this is not pretty.